runtime/reference/security_model.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "Security Model"
+---
+
+This document outlines the security model of the Deno runtime. Deno is designed
+to be secure by default runtime, however to write secure applications it is
+important to understand the security model of Deno.
+
+Deno's security model is based on the following key principles:
+
+- **No access to I/O by default**: Code executing in a Deno runtime has no
+  access to read or write arbitrary files on the file system, to make network
+  requests or open network listeners, to access environment variables, or to
+  spawn subprocesses.
+- **No limits on the execution of code at the same privilege level**: Deno
+  allows the execution of any code (JS/TS/Wasm) via multiple means, including
+  `eval`, `new Function`, dynamic imports and web workers at the same privilege
+  level with little restriction as to where the code originates (network, npm,
+  JSR, etc).
+- **Multiple invocations of the same application can share data**: Deno provides
+  a mechanism for multiple invocations of the same application to share data,
+  through built in caching and KV storage APIs. Different applications can not
+  see each other's data.
```

Comment:
Should we mention here that it's still possible with correct combination of flags to see this data?

Reply:
I think it is broken right now.